ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-defaults/advanced-gas-fee-defaults.js

```diff
@@ -16,7 +15,7 @@ import { setAdvancedGasFee } from '../../../../store/actions';
 import { useGasFeeContext } from '../../../../contexts/gasFee';
 import { useAdvancedGasFeePopoverContext } from '../context';
 import { useI18nContext } from '../../../../hooks/useI18nContext';
-import { Text } from '../../../component-library';
+import { Text, Checkbox as CheckBox } from '../../../component-library';
```

Comment:
I think we can remove the alias and update the jsx component?
```suggestion
import { Text, Checkbox } from '../../../component-library';
```

Reply:
Got it. I did that just so as to follow the naming convection that already exists